api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -668,18 +668,44 @@
           <version>4.0.0/4.0.99</version>
           <code>
             <![CDATA[
+
+    volatile Map<String, Plugin> pluginMap;
+
+    protected void cleanDelegateCache() {
```

Comment:
why do we need this? Why not just call flushPluginMap? It's only a one-liner

Reply:
Then maybe inline flushPluginMap into this method?

Reply:
I'd rather avoid any breakage in the 3.x model since it's supposed to be stable.